cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/HBaseQueueAdmin.java

```diff
@@ -188,7 +179,16 @@ public void create(QueueName queueName, Properties properties) throws IOExceptio
     // not accessible by the queue region coprocessor for doing eviction.
 
     // Create the config table first so that in case the queue table coprocessor runs, it can access the config table.
-    createConfigTable(queueName);
+    try {
+      DatasetProperties configProperties = DatasetProperties.builder()
+        .add(Table.PROPERTY_COLUMN_FAMILY, Bytes.toString(QueueEntryRow.COLUMN_FAMILY))
+        .build();
+      DatasetsUtil.createIfNotExists(datasetFramework,
```

Comment:
could this be done when the dataset framework is bootstrapped? This does not seem to be the right place to do it.